tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -72,7 +73,11 @@ sub run {
 
     if (check_var('RUN_TEST_ONLY', 0)) {
         use_ssh_serial_console;
-
+        if (is_transactional) {
```

Comment:
Is it possible to turn repetitive code into a function?

Reply:
Sure, updated. Thanks